lxd/vsock/vsock_1.11.go

```diff
@@ -0,0 +1,13 @@
+// +build go1.11
+
+package vsock
+
+import (
+	"net"
+
+	"github.com/mdlayher/vsock"
```

Comment:
@stgraber @monstermunchkin do we need this anymore? or did we go with "github.com/linuxkit/virtsock/pkg/vsock"?

Reply:
Yes, this is needed. Exec fails even for Go 1.13 with the other vsock package.